src/content/add-to-app/ios/project-setup.md

```diff
@@ -104,46 +99,55 @@ directory. Changes made in your module's `.ios/`
 directory don't appear in your existing iOS project
 using the module, and might be overwritten by Flutter.
 
-Do not source control the `.ios/` directory since it's autogenerated.
+Exclude the `.ios/` directory from source control as it's autogenerated.
 Before building the module on a new machine, run `flutter pub get`
-in the `my_flutter` directory first to regenerate the `.ios/`
-directory before building iOS project using the Flutter module.
+in the `my_flutter` directory first.
+This regenerates the `.ios/` directory before building iOS project
+using the Flutter module.
 :::
 
-## Embed the Flutter module in your existing application
+## Embed a Flutter module in your iOS app
 
 After you have developed your Flutter module,
 you can embed it using the methods described at the top of the page.
 
-:::note
 You can run in Debug mode on a simulator or a real device,
-and Release on a real device. Learn more about
-[Flutter's build modes][build modes of Flutter]
+and Release on a real device.
 
-To leverage Flutter debugging functionality 
-such as hot reload, see  [Debugging your add-to-app module][].
+Using Flutter [increases your app size][].
+
+:::note
+Learn more about [Flutter's build modes][build modes of Flutter].
+
+To use Flutter debugging features such as hot reload,
+consult [Debugging your add-to-app module][].
 :::
 
-Using Flutter [increases your app size][].
+### Method A - Embed with CocoaPods and the Flutter SDK {:#method-a}
+
+#### Requirements {:#method-a-reqs}
+
+Every developer working on your project must have a local version
+of the Flutter SDK and CocoaPods installed.
 
-### Option A - Embed with CocoaPods and the Flutter SDK
+#### Approach {:#method-a-approach}
+
+This first method compiles the Flutter module from source each time
+Xcode builds the app. To run the script to embed your Dart and plugin code,
+build your iOS app in Xcode.
```

Comment:
Should we drop the second sentence? It's original purpose was to reveal how the sausage was made. Perhaps it would be better to simplify this further?

```suggestion
This first method compiles your Dart and plugin code from source each time
Xcode builds the app.
```